python/paddle/fluid/tests/unittests/mkldnn/test_mul_mkldnn_op.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import paddle.fluid.core as core
+import numpy as np
+from paddle.fluid.tests.unittests.test_mul_op import TestMulOp, TestMulOp2
+'''
+ test case for s8 * s8
```

Comment:
There  should be s8*s8, u8*u8, s8*u8, u8*s8 four types, and each type test `TestMulOp` and `TestMulOp2`?